x/oracle/keeper/msg_server.go

```diff
@@ -15,3 +18,19 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
 }
 
 var _ types.MsgServer = msgServer{}
+
+func (k Keeper) RequestBandRates(goCtx context.Context, msg *types.MsgRequestBandRates) (*types.MsgRequestBandRatesResponse, error) {
```

Comment:
why we need this msg

Reply:
this msg supports fetching band oracle price feed data through IBC with specific `requestId`